app/views/glossary_terms/_form.html.erb

```diff
@@ -5,13 +5,36 @@
                             label: :edit_glossary_term_locked.t) %>
   <% end %>
 
-  <%= text_field_with_label(form: f, field: :name,
-                            label: :glossary_term_name.t + ":",
-                            data: { autofocus: true }) %>
+  <%=
+    text_field_with_label(
+      form: f, field: :name,
+      label: :glossary_term_name.t + ":",
+      data: { autofocus: true },
+      append: help_block(
+        :p,
+        :form_glossary_name_help.t + " " +
+        link_to(
+          :glossary_term_index_documentation.t,
+          "https://github.com/MushroomObserver/mushroom-observer/blob/main/doc/glossary.md"
+        )
+      )
+    )
+  %>
```

Comment:
@nimmolo:
Off the top of your head, do you see a way to simplify this? 
(It displays fine, so I don't think it's worth your time to dig into it.
Part of the problem is that the link is to an external file (even though it's in the repo) 
in order to make it render with GitHub flavored markdown.)


but the erb is c

Reply:
This seems right to me. Elsewhere sometimes I define the `help_block` at the top of the page as a variable to shorten up the `text_field_with_label` but it's not necessary.

Reply:
Might make more sense to move the file to a deployed folder (i'm finding public/doc for example) and call the file with a relative url:
`public/doc/glossary.md'

Reply:
Whoops — of course it's an `href` attribute. 
It needs the full path, but you can maybe use something like

```
File.join(Rails.root, "/public/doc/glossary.md")
```

Reply:
Thanks for the feedback!
The file will be deployed. I'll give that a try.
But I wonder how it will render. It's written in Github flavored markdown.

Reply:
Can't figure out how to link to the deployed file.
So sticking with what I've got for a while.
Maybe a longer term solution is to route everything through the `info` controller.

Reply:
Joe -  finally got my localhost to boot and now I understand your idea, makes sense, of course!

Github's hosted version of the file displays legible HTML, i.e. it parses the markdown. If _we_ host the file it will just display raw text, as you'd see it in a markdown editor. 

Yes, I agree that the better long-term solution is to just make a static page in the app, not necessarily under the Info controller. 

The main question is db and admin-user-interface design. It would be nice to have a standard UI for editing page content for any page in the Info controller and storing that content in the DB by page ID, so that the page "view" would have no hardcoded HTML but just render the Textile from that db entry, a textile string. 

We'd just write a simple info/edit form with a big Textile textarea, as we would for an observation description, for example — the form for the info page would pull the current text from the db, and the update route would store edits back in the database. The live page would parse the Textile as we do elsewhere.

Obviously it's less of a project to make the Glossary page with raw HTML, like other info pages. But, I understand why you didn't do that, it's not very admin friendly - it's much nicer to edit in Markdown or Textile, isn't it. 

(Fwiw, I found a path that works, so you could theoretically host the file under `/public` in the deployed file, rather than linking back to Github.)

Reply:
Aha. I think i am getting closer to a solution.

Maybe what would be useful is an Info **model**.
It could be a really simple model at first, with just a title and text. 

Then the info controller would have forms, accessible to admins only, controlling the CRUD of these info pages, and there could be routes for other namespaced InfoPages like names/info, for general info about names, or locations/info

I feel certain there are examples of stuff like this in the wild. 
This is basically what a CMS is, like WordPress. It's a very useful pattern. 
If the route itself is an attribute stored in the db, it seems even possible to dynamically "create" a brand new info page with a new route, through such a UI. Somehow routes.rb would dynamically check each entry in the Info table (in other words, each page) and create routes for each page.